packages/components/src/components/Radio/RadioGroup.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import {
+  Root as RadioGroupRoot,
+  RadioGroupProps as RadioGroupPrimitiveProps,
+} from "@radix-ui/react-radio-group";
+import { Box } from "../../primitives/Box";
+
+export interface RadioGroupProps
+  extends Omit<RadioGroupPrimitiveProps, "nonce"> {
+  /** The individual Radio components within the group. */
+  children: React.ReactNode;
+  /** The value of the radio item that should be checked when initially rendered. Use when you do not need to control the state of the radio items. */
+  defaultValue?: string;
+  /** The reading direction of the radio group. If omitted, assumes LTR (left-to-right) reading mode. */
+  dir?: "ltr" | "rtl";
+  /** The label for the radio group. */
+  label?: string;
+  /** The name of the group. Submitted with its owning form as part of a name/value pair. */
+  name?: string;
+  /** Event handler called when the value changes. */
+  onValueChange?: (value: string) => void;
+  /** The orientation of the RadioGroup */
+  orientation?: "horizontal" | "vertical";
```

Comment:
Will this do anything to our current component right now?

Reply:
Yes. Just commenting on this to confirm it was read.

Reply:
Where we will have an orientation for this in the apps?

Reply:
I hear you but this is the DS so it should be agnostic to the apps, right?

Reply:
Sorry, I misunderstood this. It makes sense to have an orientation in the RadioGroup. 👍🏻